src/ompl/base/PlannerTerminationCondition.h

```diff
@@ -120,6 +121,32 @@ namespace ompl
 
         /** \brief Return a termination condition that will become true \e duration seconds in the future (wall-time), but is checked in a separate thread, every \e interval seconds; \e interval must be less than \e duration */
         PlannerTerminationCondition timedPlannerTerminationCondition(double duration, double interval);
+
+        /** \brief Return a termination condition that will become true as soon as the problem definition has an exact solution */
+        PlannerTerminationCondition exactSolnPlannerTerminationCondition(ompl::base::ProblemDefinitionPtr pdef);
```

Comment:
Are there planners that do not terminate with an exact solution when given an infinite amount of time? If not, then why do we need this function?